lib/Indexer/Extension/IndexerExtension.php

```diff
@@ -119,17 +117,6 @@ public function load(ContainerBuilder $container): void
         $this->registerWatcher($container);
     }
 
-    private function createReflector(Container $container): Reflector
-    {
-        $builder = ReflectorBuilder::create();
-        foreach (array_keys($container->getServiceIdsForTag(WorseReflectionExtension::TAG_SOURCE_LOCATOR)) as $serviceId) {
-            $builder->addLocator($container->get($serviceId), 128);
-        }
-        $builder->enableCache();
-
-        return $builder->build();
-    }
-
```

Comment:
Phpstan has complained that this methods has no use so I removed it. And it seems like it's fine.